### PR TITLE
ISSUE Find Minimum in Rotated Sorted Array #9 solved

### PR DESCRIPTION
**Pull request by Karnankita04**
Solved #9 issue.. On Leetcode it is now accepted.. 
image

### CODE REVIEW

File: Day-1/q1-binary-search/ques.md [hunk 1]
```diff
@@ -1,11 +1,80 @@
...
+Explanation: The original array was [11,13,15,17] and it was rotated 4 times.
+
+Constraints:
+
+n == nums.length
+1 <= n <= 5000
+-5000 <= nums[i] <= 5000
+All the integers of nums are unique.
+nums is sorted and rotated between 1 and n times.
+
```
Comment (bh-g) [inline Day-1/q1-binary-search/ques.md:41]: No need to write the question.

File: Day-1/q1-binary-search/ques.md
Comment (bh-g) [inline Day-1/q1-binary-search/ques.md:1]: Rename the file as solution.cpp and move it to Day-2/q1: Find Minimum in Rotated Sorted Array

Comment (Karnankita04) [reply Day-1/q1-binary-search/ques.md:1]: okay

Comment (Karnankita04) [reply Day-1/q1-binary-search/ques.md:1]: > Rename the file as solution.cpp and move it to Day-2/q1: Find Minimum in Rotated Sorted Array

Di is it changed? please ensure it..

Comment (bh-g) [reply Day-1/q1-binary-search/ques.md:1]: Yes, the changes are visible.
Merge conflicts and move the file to Day-2/q1: Find Minimum in Rotated Sorted Array.

**Comment by bh-g** [comment]
Refer these docs:

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

**Comment by bh-g** [comment]
Do these steps and send me a screenshot of what you see:
image

**Comment by Karnankita04** [comment]
> Do these steps and send me a screenshot of what you see, and pls ask your doubts here itself.
> 
> image

image


This shows your branch is up to date 

and I also tried using other commands from code with harry video on Resolving Merge Conflicts, then it shows 

image

**Comment by bh-g** [comment]
Which code editor are you using?

**Comment by bh-g** [comment]
@Nitya-Pasrija can you please give this a look?

**Comment by Nitya-Pasrija** [comment]
I believe the errors are arising due to consistent change in the ques.md files in all the PRs here.

**Comment by Nitya-Pasrija** [comment]
Will change up the folder sectioning for the ease of contributions

**Comment by Nitya-Pasrija** [comment]
The conflict issues will be gone once you do a git pull now, also do follow the steps provided in the ques.md files and raise another PR :)

**Pull request closed without merging**